scan.go

```diff
@@ -110,12 +110,24 @@ func convertAssign(src, dest interface{}) error {
 		case *float64:
 			*dest = float64(src)
 			return nil
+		case *int64:
```

Comment:
All the integers are already covered by the generated `convertInteger` function, so we cannot do that again here.

Reply:
This is used by convert float32/64 to int64/uint64. e.g some fields like `download_rate` is shown as a float in RPC response, but is actually an integer in lib BT.

Reply:
If coder wants to use an integer it will be allowed, just change type.